src/components/Search.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Search({ onSearch }) {
+  const handleInputChange = ({ target }) => {
+    const { value } = target;
+    onSearch(value);
+  };
+
+  return (
+    <div>
+      <form>
+        <input type="text" placeholder="County name" onChange={handleInputChange} />
```

Comment:
- Hey, great job adding the search functionality, however, let's improve the design by moving it close to the container with the class of countries-grid to improve your design:


*your design*
![image](https://github.com/toyman640/metrics-webapp-capstone/assets/59397159/d1d659d9-901f-43b6-a274-4fb83ee9b181)

*main design*
![image](https://github.com/toyman640/metrics-webapp-capstone/assets/59397159/88e0a6fa-aca5-47e5-923c-b3925e54ed89)